src/loaders/index.js

```diff
@@ -0,0 +1,72 @@
+import { openArray } from 'zarr';
+// eslint-disable-next-line import/extensions
+import { fromUrl } from 'geotiff/dist/geotiff.bundle.min.js';
+
+import ZarrLoader from './zarrLoader';
+import TiffPyramidLoader from './tiffPyramidLoader';
+import { range } from '../layers/microscopy-viewer-layer/utils';
+
+async function createZarrPyramid({
```

Comment:
Is there an advantage to having these be outside the loaders?  I imageined the loaders being instantiated directly when you talked about this.

Reply:
Agreed. I think we should export `TiffPyramidLoader` and `ZarrLoader`. Right now the logic for parsing metadata is still somewhat hard-coded in the `initPyramidLoader` function. `initPyramidLoader` mostly serves as a helper function for our current demo but I think should be eliminated in the library in favor of exporting the classes directly.